andela/phlox-ah-frontend
# #160451460 Users should be able to rate articles

What does this PR do?
This PR adds a star rating to an article

Description of Task to be completed?
- Display rating average with 5 stars
- Add rating through the stars displayed
#### How should this be manually tested?
Go to the et single article url, you should see a page showing an article with it's rating average
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#160451460
